book/src/prefab.md

```diff
@@ -1,6 +1,6 @@
 # Prefabs
 
-A `Prefab` in `Amethyst` is at the core a simple list of future entities, where each entry in 
```

Comment:
Finally, thank you! Either Amethyst or `amethyst`, not `Amethyst`. Biggest issue in the book resolved xD

Reply:
We should test for this on Travis and make the build fail if somebody wrote `Amethyst` somewhere.